src/index.html

```diff
@@ -1,13 +1,434 @@
+                  <a href="#" class="link">Scarlett Hawkins</a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Senior Vice President
+                </div>
+
+                <p class="testimonial__about">
+                  My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+                </p>
+              </div>
+            </div>
+
+            <div class="testimonials__item grid__item grid__item--9-12">
+              <div class="testimonial">
+                <div class="testimonial__photo-container">
+                  <img
+                  src="images/people/3.jpg"
+                  alt="Leslie Mckinney"
+                  class="testimonial__photo"
+                  >
+                </div>
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Max Henry</a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/vGeYgx5t6ELzZb5Q6"
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN
+                  <br>
+                  55401
+                </a>
```

Comment:
target="_blank"

Reply:
Also you may wanna change color onHover, right now it mixes with background color, what looks not that great: https://prnt.sc/1tx5ph1